saviSimulation/savi/asl/demo.asl

```diff
@@ -6,3 +6,4 @@
 
 //{include("lostInThought.asl")}
 {include("working.asl")}
+//{include("multiAgent.asl")}
```

Comment:
this being commented out might explain why Cristina didn't see any changes in the behaviour

Reply:
I uncommented it, and there is anything that I can appreciate from the behavior

Reply:
isn't anything?

Reply:
Looking at simulation "video", we do not see anything different.

Reply:
There is some code cleanup as well as the beginning of some functionality needed for multi agent. On the surface, you won't be able to observe much. The changes are behind the scenes, and more of interest to @MikeVezina .